.ruby-version

```diff
@@ -1 +1 @@
-jruby-9.4.9.0
+jruby-9.4.13.0
```

Comment:
```suggestion
jruby-9.4.9.0
```
Can we handle this separately in https://github.com/elastic/logstash/pull/17798? That way i can do an easy backport.

Reply:
🤦🏼 yep, I cut that out, waiting once more for green.